README.md

```diff
@@ -31,18 +28,16 @@ Note that this uses the API in the same way as [https://speedrun.com]. The v2 AP
 When working with auth, it is recommended to construct your own `SpeedrunComPy` object rather than use the default:
 
 ```python
-from speedruncompy.endpoints import GetGameLeaderboard2
-from speedruncompy.api import SpeedrunClient
-import speedruncompy.api as srcapi
+import os
+from speedruncompy import SpeedrunClient, GetSession, set_default_PHPSESSID
 
-client = SpeedrunClient("my_app_name")
-client.PHPSESSID = "secret PHPSESSID"  # You should store this separately!
+client = SpeedrunClient("my_app_name", PHPSESSID=os.getenv("PHPSESSID")) # You should store this separately!
```

Comment:
To be beginner friendly (assuming this lib is being used by disgruntled mods more than programmers :p) it's probably worth noting what os.getenv() does & separate it out for clarity.

```suggestion
secret = os.getenv("PHPSESSID")
# You shouldn't store PHPSESSID in the script directly, instead load it externally.
# `os.getenv()` assumes you have already set env variable PHPSESSID in your terminal;
# you can load it from a file instead using `open()` or library `load_dotenv`.

client = SpeedrunClient("my_app_name", PHPSESSID=secret)
```